src/components/KarakterComponent.jsx

```diff
@@ -0,0 +1,174 @@
+import React from "react";
+import styled from "styled-components";
+import karakterImg from "../shared/assets/pic_nicolet.jpg";
```

Comment:
deze image is best groot en duurde zelfs op mijn desktop pc lang om te laden terwijl hij slechts heel klein wordt getoond. Wat kun je doen om deze te verkleinen zonder zichtbaar kwaliteitsverlies?

Reply:
FYI: hij id 4.4 MB

Reply:
ik heb https://compressjpeg.com/ gebruikt en hij is nu 942KB